strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/MavenArtifactCoordinates.java

```diff
@@ -15,22 +15,21 @@
 import org.apache.maven.artifact.Artifact;
 import org.apache.maven.artifact.versioning.ComparableVersion;
 
-/**
- * @author carlspring
- */
-import org.apache.maven.artifact.versioning.ComparableVersion;
-
 /**
  * @author carlspring
  */
 @Entity
 @XmlRootElement(name = "maven-artifact-coordinates")
 @XmlAccessorType(XmlAccessType.NONE)
-@ArtifactLayout("maven")
+@ArtifactCoordinatesLayout(name = MavenArtifactCoordinates.LAYOUT_NAME, alias = MavenArtifactCoordinates.LAYOUT_ALIAS)
```

Comment:
What's the benefit of having an annotation for this?

Reply:
we should be able to fetch all Layouts and its Coordinates to be able to generate grammar. With `ArtifactLayoutLocator` we look all classes with `ArtifactCoordinatesLayout` annotation and parse all `ArtifactLayoutCoordinate` within it. So we make it during build with groovy script `aql-dynamic-grammar.groovy`.